perfkitbenchmarker/providers/gcp/flags.py

```diff
@@ -80,3 +80,12 @@
     ['sandybridge', 'ivybridge', 'haswell', 'broadwell', 'skylake'],
     'When specified, the VM will have either the specified '
     'architecture or a newer one. Architecture availability is zone dependent.')
+flags.DEFINE_integer(
+    'gcp_gpu_count', None,
```

Comment:
No - GCP is the only provider that does not have named machine types for GPUs. Therefore this is the only way to specify GPUs. Other providers have named machine types, so these flags are not relevant.